components/alerts.css

```diff
@@ -0,0 +1,24 @@
+@import url(components.css);
+
+.alerts{
+    width: 80%;
+}
+
+.alert-pri{
+    background-color: var(--text-pri);
+    color: var(--color-white);
+}
+
+.alert-suc{
+    background-color: var(--color-off-green);
+    color: var(--color-green);
+}
+.alert-war{
```

Comment:
Consistent line spacing between your style declaration blocks should be used, Prettier  (VS code extension ) might help with this.

Reply:
Done